CHARTER.md

```diff
@@ -31,13 +31,65 @@ The membership profile is…
 ### 2.2 Community Feedback
 
 
-## 3. Meetings
+## 3. Communication
 
-### 3.1 Cadence
+### 3.1 Channels
 
-### 3.2 Annual Activities 
+Asynchronous public communication is preferred for working group activities. This section attempts to identify the communication channels along with recommendations for how and when we'll use them.
 
-Ex: budget/finance, meeting planning, elections, etc…
+*Mailing List*
+- monthly newsletter summarizing working group activities
+- used by managers for any working group announcements
+- low volume
+
+*GitHub*
+- **GitHub Discussions** used for working group feedback, suggestions, and polls
+- **GitHub Issues** used to track working group tasks
+- **GitHub Projects** used to track multiple tasks into an outcome-driven kanban aligned with working group strategic goals
+- **GitHub Pull Requests** provide opportunity for feedback from participants on any working group operating document changes
+
+*Discord*
+- use the #resource-aggregation channel for working group specific chat
+- agenda and summary of any working group calls
+- higher frequency of updates to call attention to activity from other channels
+
+*Zoom*
+- see the next section for more about working group calls
+
+### 3.2 Working Group Calls
+
+Real-time communication will utilize the community calendar to announce events to be held using Zoom for teleconferencing.
+
+*Agenda*
+- agenda is set by managers and shared a reasonable amount of time ahead of a call in Discord along with a call for topics from the community
+
+*Minutes*
+- the agenda and meeting minutes should be recorded in the GitHub repository by managers within a reasonable amount of time following the call
+
+*Schedule*
+- typically will occur immediately following manager calls on the first Friday of each month
+- two separate sessions will repeat the same agenda, one for the AMER region and one for APAC/EMEA
+- see the community calendar for working group schedule for your local timezone
+
+You can find events from the Community Calendar here:
+https://lists.dev-rel.org/g/community/calendar
+
+
+### 3.3 Annual Activities 
+
+To support the activities of the working group, there are activities that are coordinated throughout the year.
+
+*Q1*
+- charter review for community feedback and input on the working group itself, including amendments to the charter
+
+*Q2* 
+- software tools community survey and report
+
+*Q3*
+- developer persona community survey and report
+
+*Q4*
+- events community survey and report
```

Comment:
I am not sure we should commit on those surveys as thay are not 100% within the "Resource Aggregation" scope.

Reply:
Good topic for discussion. We can remove for now, but my thinking was that it was within scope because it is about collecting feedback on the resources being aggregated while making a focused push to gather additional resources that are missing from each category at a regular cadence.

Reply:
I've removed these for now, so only annual activity will be a charter review and discussion each Q1.